### ISSUE TITLE
Where are storing the trained model

### ISSUE THREAD

**Issue by shersoni610** [shashnkvats/Indofashionclip#1]
Hello,

Can you please explain how to store the trained model and how to use it?

**Comment by ariawoo** [comment]
Code to save the model :
torch.save({
        'epoch': epoch,
        'model_state_dict': model.state_dict(),
        'optimizer_state_dict': optimizer.state_dict(),
        'loss': total_loss,
        }, f"model_checkpoint/model_10.pt") #just change to your preferred folder/filename


Code to load the saved model :
model, preprocess = clip.load("ViT-B/32",device=device,jit=False) #Must set jit=False for training
checkpoint = torch.load("model_checkpoint/model_10.pt")
model.load_state_dict(checkpoint['model_state_dict'])

For more information, @shersoni610  you can visit here.  https://github.com/openai/CLIP/issues/83

**Comment by rajasblack** [comment]
How about evaluation and inference? TIA @Coffeyliu
